Makefile

```
@@ -4,16 +4,20 @@ DEVTAG=$(DRIVER_VERSION)-dev
 NAME=oras-csi-plugin
 DOCKER_REGISTRY=ghcr.io/converged-computing
 
-ready: clean compile
-publish: uninstall compile build push install
-dev: uninstall compile build-dev push-dev install
+.PHONY: help
+help: ## Generates help for all targets
+	@grep -E '^[^#[:space:]].*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-uninstall:
+ready: clean compile ## Clean pods and compiles the driver
+publish-dev: clean compile build-dev push-dev  ## TODO 
```

```suggestion
publish-dev: clean compile build-dev push-dev  ## Rebuild and publish development image
```
I'm good to rename `dev` to `publish-dev` but we will need to update that one reference to `make dev` in the README to `make publish-dev`. And if this is something we want to regularly run to develop, do you think `make dev` is not descriptive enough? I'm good with either, let's just make sure it's the same in both places.